package.json

```diff
@@ -46,21 +46,30 @@
     "module-alias": "^2.2.2",
     "morgan": "^1.9.1",
     "next": "^9.1.1",
+    "pg": "^7.12.1",
+    "pg-hstore": "^2.3.3",
     "react": "^16.10.2",
     "react-dom": "^16.10.2",
+    "reflect-metadata": "^0.1.13",
+    "sequelize": "^5.21.1",
+    "sequelize-typescript": "^1.0.0",
     "styled-components": "^4.4.0",
     "ts-node": "^8.4.1",
     "typescript": "^3.6.4"
   },
   "devDependencies": {
     "@testing-library/react": "^9.3.0",
+    "@types/bluebird": "^3.5.28",
     "@types/express": "^4.17.1",
     "@types/jest": "^24.0.19",
     "@types/morgan": "^1.7.37",
     "@types/next": "^8.0.6",
-    "@types/node": "^12.11.1",
+    "@types/node": "^12.11.2",
+    "@types/pg": "^7.11.2",
     "@types/react": "^16.9.9",
     "@types/react-dom": "^16.9.2",
+    "@types/styled-components": "^4.1.19",
+    "@types/validator": "^10.11.3",
```

Comment:
added but unused (we dont have the normal validator package anyway)

Reply:
Sequelize has it as a peer dependency, so we do use it, but it's just not in our dependencies